go.mod

```diff
@@ -1,6 +1,8 @@
 module helm.sh/helm/v3
 
-go 1.22.0
+go 1.22.4
```

Comment:
We generally only bump the Go version as required for newer kubernetes libs, for required security fixes. Check git history for examples. This message from a previous upgrade explains why:

> The version specified in the go.mod file is the minimum version we expect Helm
to be compiled against. This is the oldest supported version to support environments where others compile Helm.